worlds/cv64/aesthetics.py

```diff
@@ -214,7 +214,7 @@
           "\"Banshee Boomerang.\"",
     0x10: "No weapon triangle\n"
           "advantages with this.",
-    0x12: "It looks sus? Trust me,"
+    0x12: "It looks sus? Trust me,\n"
```

Comment:
I approved of almost all the other "space in strings" PR's, but this looks like it would get sent to the game/added to the rom, so I think the world maintainer should approve this one.

nevermind